commons/c2cgeoportal_commons/lib/email_.py

```diff
@@ -83,6 +83,8 @@ def send_email(
         smtp = smtplib.SMTP(smtp_config["host"])
     if smtp_config.get("starttls", False):
         smtp.starttls()
+    else:
+        smtp.ehlo()
```

Comment:
For me, it's strange to do that only if we don't use TLS, do you have some information about this condition or should it be like:
```suggestion
    if smtp_config.get("ehlo", True):
        smtp.ehlo()
```
And add `ehlo` definition in the `CONT_schema.yaml`.
What do you think?